src/Owin.Security.Providers.SlackProvider/Provider/SlackAuthenticatedContext_Bot.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json.Linq;
+
+namespace Owin.Security.Providers.Slack.Provider
+{
+    /// <summary>
+    /// Maintains bot data when the bot scope was requested.
+    /// </summary>
+    public class SlackAuthenticatedContext_Bot
+    {
+        public SlackAuthenticatedContext_Bot(){
+
+        }
+        public SlackAuthenticatedContext_Bot(JObject botnode)
```

Comment:
Can you please have this file accept just the userId and AccessToken in the constructor, and not have it also be responsible for parsing the jobject?